hw/cdc/tools/verixcdc/run-cdc.tcl

```diff
@@ -111,13 +118,24 @@ set_user_cntl_synchronizer -name opentitan_2ff $prim_2ff_modules
 # There are two such paths: One path is a the error pin directly into ibex, and the other is ibex's internal check
 set async_fifo_data [get_pins -of_objects [get_cells -hier * -filter {ref_name == prim_fifo_async}] -filter {name =~ rdata_o*}]
 set_ignore_cdc_paths -name async_fifo_to_ibex_data_err -through_signal $async_fifo_data -through_signal [get_pins top_earlgrey/u_rv_core_ibex/u_core/data_err_i]
-set_ignore_cdc_paths -name async_fifo_to_ibex_ecc_err -through_signal $async_fifo_data -through_signal [get_pins top_earlgrey/u_rv_core_ibex/u_core/u_ibex_core/load_store_unit_i/load_intg_err_o]
+set_ignore_cdc_paths -name async_fifo_to_ibex_ecc_err -through_signal $async_fifo_data -through_signal [get_pins top_earlgrey/u_rv_core_ibex/u_core/u_ibex_core/load_store_unit_i/load_resp_intg_err_o]
+
+# these paths are to ignore new errors due to recent design change in #15766
+# input rspfifo rd_port is qualified so we can assume its output on main_clk domain
+set_ignore_cdc_paths -name async_fifo_to_ibex -through_signal $async_fifo_data -through_signal [get_pins top_earlgrey/u_rv_core_ibex/u_core/data_rdata_i]
```

Comment:
@jeoongp in your current run where you're experimenting with RI's suggestion of `set_data_domain_clock`.. if it does not work for whatever reason, could you still leave a comment here regarding that so we make sure to revisit?